src/bentoml/_internal/cloud/schemas/schemasv2.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+import typing as t
+
+import attr
+
+from bentoml._internal.cloud.schemas.modelschemas import AccessControl
+from bentoml._internal.cloud.schemas.modelschemas import ApiServerBentoFunctionOverrides
+from bentoml._internal.cloud.schemas.modelschemas import BentoConfigServiceSchema
+from bentoml._internal.cloud.schemas.modelschemas import BentoRequestOverrides
+from bentoml._internal.cloud.schemas.modelschemas import DeploymentStrategy
+from bentoml._internal.cloud.schemas.modelschemas import DeploymentTargetConfigV2
+from bentoml._internal.cloud.schemas.modelschemas import DeploymentTargetHPAConf
```

Comment:
We can delete `cpu`, `memory`, `gpu`, `qps` in `DeploymentTargetHPAConf` right now.